std/int128.d

```diff
@@ -263,13 +263,19 @@ public struct Int128
         return this == op2 ? 0 : gt(this.data, op2.data) * 2 - 1;
     }
 
-    /** support signed arithmentic comparison operators < <= > >=
```

Comment:
Why is this deleted? Also, the previous overload still contains "signed" in its documentation.

Reply:
Because now this is a template function that handles both signed and unsigned comparison. The previous overload is for Int128 only so it's still signed.